content/source/docs/cloud/api/variable-sets.html.md

```diff
@@ -0,0 +1,247 @@
+
+`data.relationships.workspaces`          | array          |         | Array of references to workspaces that the variable set should be assigned to
+`data.relationships.vars`                | array          |         | Array of complete variable definitions that comprise the variable set.
+
+Example Payload:
+```json
+{
+  data: {
+    type: "varsets",
+    attributes: {
+      name: "varset-7f7b6783",
+      description: "Full of vars and such for mass reuse",
+      is-global: false
+    },
+    relationships: {
+      workspaces: {
+        !! Sending an empty workspaces list is an explicit declaration to unassign from all workspaces
+        data: [
+          {
+            id: "ws-z6YvbWEYoE168kpq",
+            type: "workspaces"
+          },
+          ...
+        ]
+      },
+      vars: {
+        data: [
+          {
+            type: "vars",
+            attributes: {
+              key: "c2e4612d993c18e42ef30405ea7d0e9ae",
+              value: "8676328808c5bf56ac5c8c0def3b7071",
+              category: "terraform"
+            }
+          },
+          ...
+        ]
+      }
+    }
+  }
+}
+```
+
+responds with JSON representation of the save Variable Set (see list Variable Set section for example)
+
+## Update a Variable Set
+`PUT/PATCH varsets/:external_id`
+
+rsponds with JSON representation of the saved Variable Set (see list Variable Set section for example)
+
+## Delete a Variable Set
+`DELETE varsets/:external_id`
+
+on success, responds with no content
+
+## List Variable Set
+`GET organizations/:organization_name/varsets`
+
+or
+
+`GET workspaces/:workspace_external_id/varsets`
+
+or
+
+`GET varsets/:external_id` #same response sans pagination and listing
+
+example payload
+```json
+{
+  data: [
+    {
+      id: "varset-mio9UUFyFMjU33S4",
+      type: "varsets",
+      attributes:  {
+         name: "varset-b7af6a77",
+         description: "Full of vars and such for mass reuse",
+         "is-global": false,
+         "updated-at": "2021-10-29T17:15:56.722Z",
+         "var-count":  5,
+         "workspace-count": 2
+      },
+      relationships: {
+        organization: {
+         data: {id: "organization_1", type: "organizations"}
+        },
+        vars: {
+          data: [
+           {id: "var-abcd12345", type: "vars"},
+           {id: "var-abcd12346", type: "vars"},
+           {id: "var-abcd12347", type: "vars"},
+           {id: "var-abcd12348", type: "vars"},
+           {id: "var-abcd12349", type: "vars"}
+          ]
+        },
+        workspaces: {
+          data: [
+           {id: "ws-abcd12345", type: "workspaces"},
+           {id: "ws-abcd12346", type: "workspaces"}
+          ]
+        }
+      }
+    },
+    {id: "varset-l1k3j41nf24jas",
+      ... 
+    },
+    ...
+  ],
+  links: {
+    self: "<page URL>",
+    first: "<page URL>",
+    prev: nil,
+    next: nil,
+    last: "<page URL>"
+  }
+}
+```
+
+## Relationships
+
+### Variables
+
+## Add Variable
```

Comment:
These headers are confusing me :-) Typically we have relationships at the end of the  API docs, so I was surprised to see it here. Also, there doesn't seem to be anything under the variables level 3 header?

```suggestion
## Relationships

### Variables

## Add a Variable to a Variable Set
```

Reply:
I'm still confused by this, I think!